includes/features/drupal.org/special_menu_items/special_menu_items_admin.feature

```diff
@@ -0,0 +1,58 @@
+Feature: special_menu_items administration module
+# https://www.drupal.org/project/special_menu_items
+  In order to ensure the site is functioning as expected
+  As administrator
+  I want to verify functionality of the special_menu_items module
+
+  Background:
+    Given I am logged in as a user with the "administrator" role
+
+  @safe @api
+  Scenario: Verify special_menu_items module is enabled
+    Given I am on "admin/modules"
```

Comment:
You can replace this chunk with `given the special_menu_items module is enabled`.